### PR TITLE
#11243 Moved documents and asssignments to project option

### PR DESCRIPTION
**Pull request by daniel-jeremia**
* Moved `documents` and `asssignments` to `project` option
* Updated Readme.md

### CODE REVIEW

File: quickstart/doc-based/config/config.json [hunk 1]
```diff
@@ -16,16 +16,16 @@
...
-    "source": "local",
-    "path": "quickstart/doc-based/document"
-  },
-  "assignment": {
-    "source": "local",
-    "path": "quickstart/doc-based/config/assignment.json"
-  },
   "project": {
     "teamId": "<TEAM_ID>",
+    "documents": {
```
Comment (jonathancesario) [inline quickstart/doc-based/config/config.json:21]: files?

File: quickstart/token-based/config/config.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "datasaur": {
-    "host": "https://datasaur.ai",
-    "clientId": "<DATASAUR_CLIENT_ID>",
-    "clientSecret": "<DATASAUR_CLIENT_SECRET>"
+    "host": "http://localhost:8080",
```
Comment (jonathancesario) [inline quickstart/token-based/config/config.json:3]: revert this :sweat_smile:

File: quickstart/token-based/config/config.json [hunk 3]
```diff
@@ -11,21 +11,21 @@
   "export": {
     "source": "local",
     "prefix": "quickstart/token-based/export",
-    "teamId": "<TEAM_ID>",
+    "teamId": "1",
```
Comment (jonathancesario) [inline quickstart/token-based/config/config.json:14]: revert

File: quickstart/token-based/config/config.json [hunk 4]
```diff
@@ -11,21 +11,21 @@
...
-    "source": "local",
-    "path": "quickstart/token-based/documents"
-  },
-  "assignment": {
-    "source": "local",
-    "path": "quickstart/token-based/config/assignment.json"
-  },
   "project": {
-    "teamId": "<TEAM_ID>",
+    "teamId": "1",
```
Comment (jonathancesario) [inline quickstart/token-based/config/config.json:20]: ditto

**Review by jonathancesario — CHANGES_REQUESTED** [review]
i've updated the task so please also handle the rename of `project` to `create` because it introduce breaking changes, might as well handle all the things so we release it as a new version

**Comment by daniel-jeremia** [comment]
> i've updated the task so please also handle the rename of `project` to `create` because it introduce breaking changes, might as well handle all the things so we release it as a new version

Oke kak 👍 @jonathancesario

**Review by jonathancesario — CHANGES_REQUESTED** [review]
also needs to be updated
- `robosaur.md`
- `readme.md`

File: readme.md [hunk 5]
```diff
@@ -77,11 +77,11 @@ Options:
 -h, --help  display help for command
 ```
 
-Robosaur will try to create a project for each folder inside the `documents.path` folder.
+Robosaur will try to create a project for each folder inside the `project.files.path` folder.
```
Comment (jonathancesario) [inline readme.md:80]: create.files.path?

File: readme.md [hunk 6]
```diff
@@ -217,10 +217,10 @@ In this part we will explain each part of the Robosaur config file. We will use
 
 1. Project creation (`create-projects`)
 
-   1. `"documents"`  
-      Where our project folders are located. A bit different from `projectState.path`, `documents.path` should be a folder path - relative or full.
+   1. `"files"`  
+      Where our project folders are located. A bit different from `projectState.path`, `project.files.path` should be a folder path - relative or full.
```
Comment (jonathancesario) [inline readme.md:221]: ditto

File: readme.md [hunk 7]
```diff
@@ -217,10 +217,10 @@ In this part we will explain each part of the Robosaur config file. We will use
...
    2. `"assignment"`  
-      Where our assignment file is located. `assignment.path` is similar to `projectState.path`, it should be a full or relative path pointing to a JSON file.  
+      Where our assignment file is located. `project.assignment.path` is similar to `projectState.path`, it should be a full or relative path pointing to a JSON file.
```
Comment (jonathancesario) [inline readme.md:223]: create.assignment.path?

**Review by jonathancesario — CHANGES_REQUESTED** [review]

File: readme.md (hunk 5 above)
Comment (daniel-jeremia) [reply readme.md:80]: eh.. iya ya udah ganti `create` xD

**Review by jonathancesario — APPROVED** [review]

**Pull request merged**